src/components/SpinButton.jsx

```diff
@@ -18,8 +18,8 @@ const StyledButton = styled.button`
   }
 `;
 
-function SpinButton() {
-  const { isSpinning, balance, messageWinner, handleSpinButton } = useSpin();
+function SpinButton(props) {
+  const { handleSpinButton, isSpinning, balance, messageWinner } = props;
   return (
     <StyledButton
       disabled={isSpinning || balance === 0 ? true : false}
```

Comment:
here you can do `disabled={isSpinning || balance >= 0)`. Inside the { } you are executing a condition. The first `isSpinning` is referencing a boolean. `balance >= 0` is the same as checking if the balance greater than or equal to 0 which returns a boolean as well.